doc/source/how-to/dns-configuration.rst

```diff
@@ -6,9 +6,9 @@ online under the following canonical name (CNAME) convention:
 
 ``https://<product>.docs.pyansys.com``
 
-To request a CNAME for the ``pyansys.com`` domain, contact
-`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
-can handle the creation of the requested PyAnsys subdomain.
+To request a CNAME for the ``docs.pyansys.com`` domain, contact the
```

Comment:
The domain is ``pyansys.com`` and ``docs.pyansys.com`` is a subdomain. Although we usually create ``*.docs.pyansys.com``, let's leave the ``pyansys.com`` reference in here.

```suggestion
To request a CNAME for the ``pyansys.com`` domain, contact the
```